CHANGELOG.md

```diff
@@ -43,7 +43,8 @@ Release Versions:
 - release: use updated base image (#139)
 - fix(controllers): remove duplicate function (#140)
 - chore: format repository (#142)
-- feat: add support for generic outputs (#133)
+- feat: add support for generic inputs/outputs (#133)
+- feat: add support for generic generic subscriptions
```

Comment:
This is just for the CI to run the tests, we should clean up the CHANGELOG in the final PR.

Reply:
The CI will always run the tests btw and they might succeed but because the changelog fails, the overall action fails

Reply:
ah yes, true